experimenter/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -1108,3 +1118,155 @@ def test_exception_for_fetch_experiment_data(self, mock_get_experiment_data):
+                                        "sample_size_per_branch": 20.0,
+                                        "population_percent_per_branch": 57.142857142
+                                    }
+                                },
+                                "parameters": {
+                                    "power": 0.8,
+                                    "effect_size": 0.05
+                                }
+                            }
+                        }
+                    },
+                    "existing": {
+                        "target_recipe": {
+                            "app_id": "firefox_desktop",
+                            "channel": "release",
+                            "locale": "('EN-US')",
+                            "country": "US",
+                            "new_or_existing": "existing",
+                            "minimum_version": "108"
+                        },
+                        "sample_sizes": {
+                            "Power0.8EffectSize0.05": {
+                                "metrics": {
+                                    "active_hours": {
+                                        "number_of_clients_targeted": 10000,
+                                        "sample_size_per_branch": 100000.0,
+                                        "population_percent_per_branch": 1000.0
+                                    },
+                                    "search_count": {
+                                        "number_of_clients_targeted": 10000,
+                                        "sample_size_per_branch": 100.0,
+                                        "population_percent_per_branch": 1.0
+                                    },
+                                    "days_of_use": {
+                                        "number_of_clients_targeted": 10000,
+                                        "sample_size_per_branch": 10000.0,
+                                        "population_percent_per_branch": 100.0
+                                    }
+                                },
+                                "parameters": {
+                                    "power": 0.8,
+                                    "effect_size": 0.05
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            """
+
+        class File:
+            def __init__(self, filename):
+                self.name = filename
+
+            def read(self):
+                if "sample_sizes" not in self.name:
+                    return ""
+                return sizing_test_data
+
+        def open_file(filename):
+            return File(filename)
+
+        mock_open.side_effect = open_file
+        mock_exists.return_value = True
+
+        sizing_results = cache.get(SIZING_DATA_KEY)
+        assert sizing_results is None
```

Comment:
So this is the nittiest of all nits, but when you're in a `unittest` context you have all the [built in assertions](https://docs.python.org/3/library/unittest.html#unittest.TestCase.debug) which will format the output nicely, whereas `assert` is really naive and will just throw an exception, so generally if you're in a `unittest` context it's nicer to use the builtin asserts.

Reply:
I'm not sure why I didn't use `assertIsNone` here! Good catch.